app/src/features/admin/users/UsersDetailHeader.tsx

```diff
@@ -0,0 +1,197 @@
+    dialogContext.setYesNoDialog({
+      ...defaultYesNoDialogProps,
+      ...yesNoDialogProps,
+      open: true
+    });
+  };
+
+  const openErrorDialog = useCallback(
+    (errorDialogProps?: Partial<IErrorDialogProps>) => {
+      dialogContext.setErrorDialog({
+        ...defaultErrorDialogProps,
+        ...errorDialogProps,
+        open: true
+      });
+    },
+    [defaultErrorDialogProps, dialogContext]
+  );
+
+  const handleDeleteUser = async (userId: number) => {
+    const apiCall = await biohubApi.project.getAllUserProjectsForView(userId);
+
+    for (const project of apiCall) {
+      const response = await biohubApi.project.getProjectParticipants(project.project_id);
+      const projectLeadAprroval = checkForProjectLead(response, project.project_participation_id);
+
+      if (!projectLeadAprroval) {
+        openErrorDialog({
+          dialogTitle: SystemUserI18N.deleteProjectLeadErrorTitle,
+          dialogText: SystemUserI18N.deleteProjectLeadErrorText
+        });
+        return;
+      }
+    }
+    deActivateSystemUser(userDetails);
+  };
+
+  const deActivateSystemUser = async (user: IGetUserResponse) => {
+    if (!user?.id) {
+      return;
+    }
+    try {
+      await biohubApi.user.deleteSystemUser(user.id);
+
+      dialogContext.setSnackbar({
+        snackbarMessage: (
+          <>
+            <Typography variant="body2" component="div">
+              User <strong>{user.user_identifier}</strong> removed from application.
+            </Typography>
+          </>
+        ),
+        open: true
+      });
+
+      uHistory.push('/admin/users');
+    } catch (error) {
+      const apiError = error as APIError;
+      openErrorDialog({ dialogText: apiError.message, dialogErrorDetails: apiError.errors, open: true });
+    }
+  };
+
+  return (
+    <Paper square={true}>
+      <Container maxWidth="xl">
+        <Box pt={3} pb={2}>
+          <Breadcrumbs>
+            <Link
+              color="primary"
+              onClick={() => uHistory.push('/admin/users')}
+              aria-current="page"
+              className={classes.breadCrumbLink}>
+              <Typography variant="body2">Manage Users</Typography>
+            </Link>
+            <Typography variant="body2">{userDetails.user_identifier}</Typography>
+          </Breadcrumbs>
+        </Box>
+        <Box display="flex" justifyContent="space-between" alignItems="flex-start">
+          <Box pb={3}>
+            <Box mb={1.5} display="flex">
+              <Typography className={classes.spacingRight} variant="h1">
+                User - <span className={classes.projectTitle}>{userDetails.user_identifier}</span>
+              </Typography>
+            </Box>
+            <Box mb={0.75} display="flex" alignItems="center">
+              <Typography component="span" variant="subtitle1" color="textSecondary">
+                &nbsp;&nbsp;{userDetails.role_names[0]}
```

Comment:
Are the 2 spaces needed? Probably just remove.  If an indent is desired then we can add some margin or padding instead. (but as always, whatever Jeremy says goes whenever it comes to UI stuff)